ProjectSidewalk/SidewalkWebpage
# #2150 logging in tutorial

Resolves checkboxes 2 through 5 of #2150 

Limits logging from 10*2^n to 2n (ish better at times worse at others) in the tutorial as well as removes audit id  from the "ContextMenu_TagAdded" action.

##### Testing instructions
1.  go through tutorial
2. check the database for logs that apply

EDIT: UPDATE FROM MIKEY 7/23/2021
1. I've fixed things so that checkboxes 2 through 6 of #2150 have been fixed. (Fixes #2150)
2. I've added a "step" variable to the set of notes associated with `OnboardingTransition` logs, which should make it easier to figure out how far someone got in the tutorial during data analysis.
3. I renamed the `rate-attribute-1` log to `rate-severity-1` to match the format for the other labels.
